docs/iceberg-rest-service.md

```diff
@@ -130,16 +130,22 @@ The Gravitino Iceberg REST catalog service uses the memory catalog backend by de
+| `gravitino.iceberg-rest.uri`                                              | The JDBC connection address, such as `jdbc:postgresql://127.0.0.1:5432` for Postgres, or `jdbc:mysql://127.0.0.1:3306/` for mysql.                                                                                                                              | (none)                   | Yes                                                                                | 0.2.0         |
+| `gravitino.iceberg-rest.warehouse `                                       | The warehouse directory of JDBC catalog. Set the HDFS prefix if using HDFS, such as `hdfs://127.0.0.1:9000/user/hive/warehouse-jdbc`                                                                                                                            | (none)                   | Yes                                                                                | 0.2.0         |
+| `gravitino.iceberg-rest.catalog-backend-name`                             | The catalog name passed to underlying Iceberg catalog backend. Catalog name in JDBC backend is used to isolate namespace and tables.                                                                                                                            | `jdbc` for JDBC backend  | No                                                                                 | 0.5.2         |
+| `gravitino.iceberg-rest.authentication.type`                              | The type of authentication for Iceberg rest catalog backend. This configuration only applicable for for Hive and REST backend, and only supports `Kerberos`, `simple` currently. As for JDBC backend, only username/password authentication was supported now.  | `simple`                 | No                                                                                 | 0.6.0         |
+| `gravitino.iceberg-rest.authentication.impersonation-enable`              | Whether to enable impersonation for the Iceberg catalog                                                                                                                                                                                                         | `false`                  | No                                                                                 | 0.6.0         |
+| `gravitino.iceberg-rest.authentication.kerberos.principal`                | The principal of the Kerberos authentication                                                                                                                                                                                                                    | (none)                   | required if the value of `gravitino.iceberg-rest.authentication.type` is Kerberos. | 0.6.0         |
```

Comment:
the security configuration should place int `security` part

Reply:
I added authentication configurations to the Hive backend part.